### PR TITLE
Finnish dictionary and language package

### PR DESCRIPTION
**Pull request by sasxa**

### CODE REVIEW

File: packages/languages/fi/README.md [hunk 1]
```diff
@@ -0,0 +1,39 @@
+# @zxcvbn-ts/language-fi
+
+The Finnish dictionary and language package for zxcvbn-ts
+
+Data sources for first and last names:
+
+    https://www.avoindata.fi/data/fi/dataset/none
```
Comment (MrWook) [inline packages/languages/fi/README.md:7]: If the name list are from this site could you directly use them please? There is already an ExcelGenerator for the nl-be language that you can use too

File: packages/languages/fi/README.md [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+#### yarn:
+
+`yarn add @zxcvbn-ts/language-fi`
+
+## Setup
+
+```js
+import { zxcvbn, zxcvbnOptions } from '@zxcvbn-ts/core'
+import zxcvbnCommonPackage from '@zxcvbn-ts/language-common'
+import zxcvbnItPackage from '@zxcvbn-ts/language-fi'
```
Comment (MrWook) [inline packages/languages/fi/README.md:24]: `zxcvbnFiPackage`

File: packages/languages/fi/README.md [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+const password = 'somePassword'
+const options = {
+  translations: zxcvbnItPackage.translations,
```
Comment (MrWook) [inline packages/languages/fi/README.md:28]: `zxcvbnFiPackage`

File: packages/languages/fi/README.md [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+  graphs: zxcvbnCommonPackage.adjacencyGraphs,
+  dictionary: {
+    ...zxcvbnCommonPackage.dictionary,
+    ...zxcvbnItPackage.dictionary,
```
Comment (MrWook) [inline packages/languages/fi/README.md:32]: `zxcvbnFiPackage`

**Comment by sasxa** [comment]
Thanks for the code review. I updated the lists to use ExcelGenerator and fixed typos in README.

**Comment by MrWook** [comment]
I will generate the wikipedia dictionary later and then publish this package. Thanks for your contribution 👍

**Comment by MrWook** [comment]
Is published

**Pull request merged**